about.html

```diff
@@ -0,0 +1,182 @@
+                <a class="nav-link" id="campaign-link" href="#news">TC Campaign</a>
+            </li>
+            </ul>
+        </div>
+        </div>
+    </nav>
+
+        <!-- Event details page -->
+
+    <section class="event-section">
+
+        <div class="h-text text-center">
+            <h1 class="text-left ps-0 pt-4">
+                TechCon Conference
+                Week
+              </h1>
+        </div>
+        <div class="p-text text-center justify-content-center">
+           <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Quod, omnis cumque nisi libero ullam repellat voluptatum, quasi voluptatibus voluptates quibusdam delectus quo alias itaque, accusantium eius iusto maxime blanditiis mollitia temporibus. Unde, saepe delectus. Sed tempora vero distinctio mollitia provident, odit libero voluptatem quo cupiditate, aperiam totam ipsa sunt cumque!
+           </p>
+        </div>
+    </section>
+
+    <!-- About details -->
+
+    <section>
+        <div class="about-heading text-center mt-4">
+            <h2>TechCon Conference Logo</h2>
+        </div>
+        <div class="guide_bar"></div>
+        <div class="about-description text-center">
+            <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Dolor, saepe fugiat. Ratione repellendus, aliquam omnis obcaecati, aperiam neque temporibus, laborum in repudiandae harum pariatur molestias quam! Nesciunt eius saepe unde quaerat ratione, in dicta quae iure aut eum quis nemo? Aut pariatur sint quibusdam tempore illo a dolore quo </p>
+        </div>
+        <a class="brandLogo pb-4" href="index.html"><span class="text-warning">Tech</span>Con</a>
+    </section>
+
+    <!-- See previous events -->
+
+    <section class="bg-dark w-100">
+        <div class="container">
+            <div class="text-center text-white pt-4">
+                <h2>Previous webinars</h2>
+                <div class="guide_bar"></div>
+            </div>
+            <div class="row justify-content-center pb-5 gap-4">
+                <div class="col-md-4 bg-image ripple" data-mdb-ripple-color="light">
+                    <img src="./images/virtual-pic-1.jpg" class="w-100" />
+                    <a href="#!">
+                      <div id="mask" class="mask">
+                        <div class="d-flex justify-content-center align-items-center h-100">
+                          <h4 class="text-white text-center">2021<br>
+                            TechCon in 2021</h4>
+                        </div>
+                      </div>
+                      <div class="hover-overlay">
+                        <div id="mask-2" class="mask"></div>
+                      </div>
+                    </a>
+                  </div>
+                <div class="col-md-4 bg-image ripple" data-mdb-ripple-color="light">
+                    <img src="./images/virtual-pic-2.jpg" class="w-100" />
+                    <a href="#!">
+                      <div id="mask" class="mask">
```

Comment:
# Desktop About
- **It contains a bottom section with a central column that includes a section header, a divider, a line of text, and two images with text centered over them. Kindly follow the design**
- **Please About does not contain a partner section**
- your page 
- 
![Capture](https://user-images.githubusercontent.com/63972670/198672600-cb907941-ddb7-44f4-9355-08eb5bd6dacf.PNG)
- Original page
- 
![Capture1](https://user-images.githubusercontent.com/63972670/198672721-9124b6e7-72c8-4953-a8b0-56eeae48a6b2.PNG)